src/compiler/crystal/syntax/parser.cr

```diff
@@ -315,7 +315,6 @@ module Crystal
           break
         when :"="
           slash_is_regex!
-
```

Comment:
Pedantic: That line could be left as is :)

Reply:
@Sija Done :)